frontend/src/pages/ChainMemberList.tsx

```diff
@@ -362,3 +368,6 @@ const HeadRowTableCell = ({ children, ...props }: { children?: any }) => {
 };
 
 export default ChainMemberList;
+function t(arg0: string): string {
+  throw new Error("Function not implemented.");
+}
```

Comment:
Accidental code editor "fix"?

Reply:
Possibly yes, don't think is in use anywhere. Just going to remove it.